modules/components/pages/redpanda/about.adoc

```diff
@@ -321,7 +388,99 @@ The token to use for a single session's OAUTHBEARER authentication.
 
 === `sasl[].extensions`
 
-Key/value pairs to add to OAUTHBEARER authentication requests.
+Key/value pairs to add to `OAUTHBEARER` authentication requests.
+
+
+=== `sasl[].aws`
+
+AWS specific fields for when the `mechanism` is set to `AWS_MSK_IAM`.
+
+*Type*: `object`
+
+=== `sasl[].aws.region`
+
+The AWS region to target.
+
+*Type*: `string`
+
+*Default*: `""`
+
+=== `sasl[].aws.endpoint`
+
+Specify a custom endpoint for the AWS API.
+
+*Type*: `string`
+
+*Default*: `""`
+
+=== `sasl[].aws.credentials`
+
+Manually configure the AWS credentials to use (optional). For more information, see the xref:guides:cloud/aws.adoc[Amazon Web Services guide].
+
+*Type*: `object`
+
+ifndef::env-cloud[]
```

Comment:
I've assumed this is cloud only but is this correct?

Reply:
I have removed this until the question of what should be included in the Cloud docs is resolved.